fixtures/modal.fixture.js

```diff
@@ -0,0 +1,64 @@
+import {ModalLauncher, StandardModal} from "@khanacademy/wonder-blocks-modal";
+import Button from "@khanacademy/wonder-blocks-button";
+import {Body} from "@khanacademy/wonder-blocks-typography";
+
+import {describe, expect, test, render, simulate, sleep} from "./helpers.js";
+
+describe("Modal", () => {
+    test("StandardModal", async () => {
+        const standardModal = ({closeModal}: any) => (
+            <StandardModal
+                title="Title"
+                subtitle="You're reading the subtitle!"
+                content={
+                    <Body tag="p">
+                        {
+                            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."
+                        }
+                    </Body>
+                }
+                footer={
+                    // TODO(mdr): Use Wonder Blocks Button.
+                    <Button onClick={closeModal}>Close modal</Button>
+                }
+            />
+        );
+
+        await render(
+            <ModalLauncher modal={standardModal}>
+                {({openModal}) => (
+                    <Button onClick={openModal}>Standard modal</Button>
+                )}
+            </ModalLauncher>,
+        );
+
+        // open the modal
+        await simulate({type: "mousemove", clientX: 50, clientY: 25});
+        await simulate({type: "click"});
+        await sleep(100);
+        await expect(document.body).toMatchScreenshot();
+
+        // tab repeatedly to verify that the focus trap works
+        await simulate({type: "keypress", key: "tab"});
+        await expect(document.body).toMatchScreenshot();
+        await simulate({type: "keypress", key: "tab"});
+        await expect(document.body).toMatchScreenshot();
+
+        // get the location of the close button and move the mouse over it
+        const closeButton = document.querySelector(
+            `[data-test-id="close-modal"]`,
+        );
+        const bounds = closeButton.getBoundingClientRect();
+        await simulate({
+            type: "mousemove",
+            clientX: bounds.left + bounds.width / 2,
+            clientY: bounds.top + bounds.height / 2,
+        });
```

Comment:
It would be nice to have a helper that computes the coordinates of the middle of an element.